e2e-polybft/e2e/apex_bridge_test.go

```diff
@@ -446,23 +452,62 @@ func TestE2E_ApexBridge_InvalidScenarios(t *testing.T) {
 	user := apex.Users[0]
 
 	txProviderPrime := apex.PrimeInfo.GetTxProvider()
+	txProviderVector := apex.VectorInfo.GetTxProvider()
+
+	primeNet := infrawallet.TestNetNetwork
+	vectorNet := infrawallet.VectorTestNetNetwork
+
+	txSender := sendtx.NewTxSender(
+		bridgingFeeAmount,
+		cardanofw.MinUTxODefaultValue,
+		potentialFee,
+		maxInputsPerTx,
+		map[string]sendtx.ChainConfig{
```

Comment:
can we define this chain configuration in some function or in one place. if it is used in multiple places